libs/build.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const { mkdir, rm, write, read } = require('./fileUtils');
+
+// !!WARNING!! Do not include a '.' in the directory name this confuses
+// sls during local invokes and compilers with a MODULE_NOT_FOUND error
+const BUILD_DIR = 'ignitor';
+
+const DEFAULT_WRAPPER = 'ignitor.ignitor';
+
+const prebuild = () => {
+  mkdir(BUILD_DIR);
+
+  // prebuild default wrapper into the build directory
+  const defaultWrapper = read(path.resolve(__dirname, 'ignitor.js'));
+  const defaultWrapperPath = path.resolve(BUILD_DIR, 'ignitor.js');
+  write(defaultWrapperPath, defaultWrapper);
+}
+
+const wrap = (name, handler, wrapper = DEFAULT_WRAPPER) => {
+  const [wrapperPath, wrapperFunctionName] = wrapper.split('.');
+  const [inputPath, functionName] = handler.split('.')
+
+  const overridePath = `${BUILD_DIR}/${name}.default`;
+  const overrideFilename = `${name}.js`;
+  const outputPath = path.resolve(BUILD_DIR, overrideFilename);
+
+  const requireOriginal = `const { ${functionName} } = require('../${inputPath}');`
+  const requireWrapperPrefix = wrapper === DEFAULT_WRAPPER ? '.' : '..';
+  const requireWrapper = `const { ${wrapperFunctionName} } = require('${requireWrapperPrefix}/${wrapperPath}');`;
```

Comment:
Please note requireWrapperPrefix, this will eventually be used when a custom wrapper is provided by the end-user. The `..` is because the require will be relative to the user's project dir, whereas the defaultWrapper already having been added to the buildDir only uses on `.`

Reply:
With all of the path usage going around, how about some debugging information that appears when you use -v (verbose) to assist users of the module in case it doesn't work as expected